lib/storage/storage.go

```diff
@@ -573,6 +575,54 @@ func (s *Storage) UpdateMetrics(m *Metrics) {
 	s.tb.UpdateMetrics(&m.TableMetrics)
 }
 
+var (
+	isStorageLimitReached      uint32
```

Comment:
Please rename this var to `isFreeDiskSpaceLimitReached` to be consistent with other similar names